packages/html2react/package.json

```diff
@@ -21,12 +21,14 @@
     "test:inspect": "node --inspect ../../node_modules/.bin/jest --watch --no-cache --runInBand"
   },
   "devDependencies": {
+    "@emotion/jest": "^11.2.1",
     "@frontity/connect": "^1.2.1",
     "@types/react-test-renderer": "^17.0.0",
-    "@testing-library/react": "^9.4.0",
+    "@testing-library/react": "^11.2.5",
```

Comment:
Updated to the latest version because it was throwing errors when running tests locally.